docs/performance_tuning.md

```diff
@@ -144,6 +144,38 @@ To save power, the OS can decrease the CPU frequency and increase a volatility o
 $ cpupower frequency-set --min 3.1GHz
 ```
 
+## Network Configuration for Optimal Performance
+
+When clients connect to the server using hostname resolution (particularly "localhost"), the system may attempt IPv6 resolution first before falling back to IPv4. If IPv6 is disabled, misconfigured, or unavailable, this can cause connection timeouts and delays before the IPv4 fallback occurs, which is especially noticeable when minimizing time to first token in generative AI applications.
```

Comment:
By default, OVMS endpoints are bound to all ipv4 addresses. On same systems, which route `localhost` name to ipv6 address, it might cause extra time on the client side to switch to ipv4. It can effectively results with extra 1-2s latency.
It can be overcome by switching the API URL to http://127.0.0.1  instead.

Alternatively ipv6 can be enabled in the model server using --grpc_bind_address and --rest_bind_address. 
For example:
`--grpc_bind_address 127.0.0.1,::1 --rest_bind_address 127.0.0.1,::1`
or 
`--grpc_bind_address 0.0.0.0,:: --rest_bind_address 0.0.0.0,::`

Reply:
changed